enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/rest/RestOnboardingProvider.java

```diff
@@ -53,7 +60,7 @@ public LookupUserResponse lookupUser(final LookupUserRequest request) throws Onb
         final UserLookupResponseDto response;
 
         try {
-            response = restClient.post("/user/lookup", requestDto, responseType).getBody();
+            response = restClient.post("/user/lookup", requestDto, null, createHeaders(), responseType).getBody();
```

Comment:
We may consider to log headers. But I guess it should be responsibility of underlaying `RestClient` and should be limited e.g. to `DEBUG` because it may contain sensitive information.